lib/facter/self_service.rb

```diff
@@ -108,6 +108,12 @@
     { S0013: File.open(summary_path).read.include?('catalog_application') }
   end
 
+  chunk(:S0014) do
+    file_time = File.mtime('/opt/puppetlabs/server/data/puppetdb/stockpile/cmd/q')
+    time_now = Time.now - 1800
```

Comment:
```suggestion
    time_now = Time.now - Puppet.settings['runinterval']
```

Reply:
this setting returns and integer in seconds that is more accurate for what we are looking for, the run_interval may be changed to compensate for slow processing time

Reply:
Changed to reflect this